packages/sdks/snippets/qwik-city/src/routes/live-preview/[...index]/index.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  component$,
+  useSignal,
+  useTask$,
+  useVisibleTask$,
+} from '@builder.io/qwik';
+import { routeLoader$ } from '@builder.io/qwik-city';
+import type { BuilderContent } from '@builder.io/sdk-qwik';
+import {
+  fetchOneEntry,
+  getBuilderSearchParams,
+  subscribeToEditor,
+} from '@builder.io/sdk-qwik';
+
+export const useBuilderContent = routeLoader$(async ({ url }) => {
+  const data = await fetchOneEntry({
+    model: 'blog-data',
+    apiKey: 'ee9f13b4981e489a9a1209887695ef2b',
+    options: getBuilderSearchParams(url.searchParams),
+    userAttributes: {
+      urlPath: url.pathname,
+    },
+  });
+  return data ?? null;
+});
+
+export default component$(() => {
+  const { value } = useBuilderContent();
+
+  const content = useSignal<BuilderContent | null>(null);
+  useTask$(({ track }) => {
+    const data = track(() => value);
+    if (data !== undefined) {
+      content.value = data;
+    }
+  });
```

Comment:
- `useTask$` is unnecessary here, because `useBuilderContent` will only provide a value once initially.
- you can provide it as an initial value to `useSignal`

```suggestion
  const content = useSignal<BuilderContent | null>(builderContent.value);
```